app/controllers/point_details_controller.rb

```diff
@@ -5,12 +5,15 @@ class PointDetailsController < ApplicationController
   before_action :changeable_evaluation
 
   def update
-    point_request = PointRequest.find_or_create_by(evaluation: @evaluation, user: @user)
-    @point_detail = PointDetail.find_or_create_by(point_request: point_request, principle: @principle)
-    @point_detail.update(point: valid_point)
-
-    @point_details = PointDetail.includes(%i[point_request principle]).select do |pd|
-      pd.point_request.evaluation == @evaluation && pd.point_request.user_id == @user.id
+    point_request =
```

Comment:
I think we should consider the 100 characters per line.
I remember we talked about it, but we have to agree to it. For example, in a rubocop.yml file